lib/dgram.js

```diff
@@ -151,6 +154,26 @@ function bufferSize(self, size, buffer) {
   return ret;
 }
 
+function bindServerHandle(self, options, errCb) {
+  if (!cluster)
+    cluster = require('cluster');
+
+  const state = self[kStateSymbol];
+  cluster._getServer(self, options, (err, handle) => {
+    if (err) {
+      errCb(err);
+      return;
+    }
+
+    if (!state.handle)
```

Comment:
Minor nit: can braces be included with the conditional's multi-line body?

Reply:
Done.